@here/harp-mapview/lib/MapView.ts

```diff
@@ -808,6 +832,10 @@ export class MapView extends THREE.EventDispatcher {
                 options.quadTreeSearchDistanceDown;
         }
 
+        if (options.enablePolarDataSource !== undefined) {
+            this.m_enablePolarDataSource = options.enablePolarDataSource;
```

Comment:
I think if it is not explicitly turned of I would enable it by default if projection is spherical

Reply:
the default is enabled, so it works this way right now.